packages/wrangler/src/pages/dev.tsx

```diff
@@ -541,7 +541,7 @@ export const Handler = async ({
 
 	CLEANUP_CALLBACKS.push(stop);
 
-	waitUntilExit().then(() => {
+	void waitUntilExit().then(() => {
```

Comment:
Perhaps this should just use a `.finally` instead?

Reply:
Probably, but we aren't really doing anything with the Promise